api/app/models/concerns/spree/webhooks/has_webhooks.rb

```diff
@@ -46,8 +46,8 @@ def updating_only_timestamps?
         (saved_changes.keys - %w[created_at updated_at deleted_at]).empty?
       end
 
-      def default_event?(event_name)
-        self.class.default_webhook_events.include?(event_name)
+      def update_event?(event_name)
+        event_name.include?('.update')
```

Comment:
I think there should not be a case where ".update" is in the middle of the event name, but to make the intent clearer you could use [end_with?](https://ruby-doc.org/core-3.0.3/String.html#method-i-end_with-3F).